app/components/action_button.rb

```diff
@@ -22,7 +22,7 @@ def button_data
       if confirm
         # :confirm trumps :blacklight_modal, because :blacklight_modal would negate :confirm by firing the ajax request regardless of the user's decision
         data[:turbo_confirm] = confirm
-      elsif !new_page
+      elsif open_modal
```

Comment:
I'm not saying this is a problem, but I'm observing that the default behavior before (since `new_page` defaulted to `nil`) was to trigger this condition and now the default behavior is not to trigger this condition. Intended? Not a problem if you touched all the places that call this code and do not want to trigger this condition.

Reply:
There weren't that many spots that used this (all in the component itself) and I changed them.  i just want the attribute to match the intent ("new_page" currently means no modal AND only GET requests).

in my mind, showing a modal better implies a get request, so that name seems clearer to me

Reply:
WFM